middleware/keyauth/keyauth.go

```diff
@@ -107,7 +147,7 @@ func keyFromQuery(param string) func(c fiber.Ctx) (string, error) {
 }
 
 // keyFromForm returns a function that extracts api key from the form.
-func keyFromForm(param string) func(c fiber.Ctx) (string, error) {
+func keyFromForm(param string) KeyauthKeyLookupFunc {
```

Comment:
```suggestion
func KeyFromForm(param string) KeyauthKeyLookupFunc {
```

Reply:
@dave-gray101 pls check

Reply:
Sorry, this has already been fixed as well - https://github.com/gofiber/fiber/blob/2711dc315549f2ec53b10d104d8d62dbb1adbbce/middleware/keyauth/keyauth.go#L151

I just forgot to resolve the comment